Docs/development/msix-nightly-testing.md

```diff
@@ -0,0 +1,13 @@
+# MSIX Nightly Testing
+When installing the JASP nightly you may encounter this error:
+
+![error](https://github.com/RensDofferhoff/jasp-desktop/assets/20978635/05d2499d-3928-43ae-944d-aee63dc33680)
```

Comment:
Should creat a branch for upstream and make a PR. then  assets paths should be `jasp-stats` right?

Reply:
good point! I will just merge it and do it in place :p